Makefile

```diff
@@ -0,0 +1,38 @@
+
```

Comment:
What about `make install` ? I can see the usefulness of having `mwctl` in your path (once things are stabilised, ofc)